### ISSUE TITLE
Insert Image / File Manager page is empty

### ISSUE THREAD

**Issue by markwaters** [idno/Known#1797]
## While trying to do this:
Insert Image using Rich Text Editor.
1. Click Insert / Edit Image in Rich Editor Toolbar.
2. Insert / Edit pops-up , click File Manager icon for Source.
3. Empty File Manager Window pops-up.

## I encountered this error:
Blank File Manager page.
https://drive.google.com/file/d/0B7Ob1JNjKbzsZXBqaDc0THdKSUE/view?usp=sharing

## Some other notes:
Seeing same results in Firefox 52 and Chromium 57 on Debian Linux , and Firefox 52 on Android.
When @mapkyca recently switched the default rich text editor this functionality started working correctly again , and stopped again when he switched it back.
I'm using the latest git master version , currently 2a23b484c22d078877ca11d19f13761fe13317e6
version.known = version = "0.9.9" build = 2017062701

**Comment by mapkyca** [comment]
<a href="https://github.com/idno/Known/issues/1797" class="u-in-reply-to"></a>Not seeing this (although I am seeing a separate issue to do with image previews which I'll investigate).

Could you check to see if there are any errors in your dev console? Specifically I'm wondering if there's some unintentional cross domain stuff going on which is preventing the iframe from inserting.

**Comment by markwaters** [comment]
Thanks @mapkyca , when I bring up Developer / Browser Console in Firefox and then perform the steps above I see this error.
Load denied by X-Frame-Options: https://markwaters.eu/filepicker/?type=image does not permit framing.

**Comment by mapkyca** [comment]
<a href="https://github.com/idno/Known/issues/1797" class="u-in-reply-to"></a>Sounds like there's a discrepency between the URL you're visiting and the display URL...

**Comment by mapkyca** [comment]
<a href="https://github.com/idno/Known/issues/1797" class="u-in-reply-to"></a>... Are you running this from your markwaters.eu domain or linking to it from somewhere else?

I've tried to find it on your site but got redirected to about.me....

**Comment by markwaters** [comment]
I have upgraded to a15a7123c4de9e601013866e76b57ffb6c97dc03

The problem is still happening in Firefox and Chrome under Linux and Firefox on Android.

When I use Microsoft Edge on Windows 10 to bring up the File Manager page for uploading an image I see this error message

This content can’t be shown in a frame
There is supposed to be some content here, but the publisher doesn’t allow it to be displayed in a frame. This is to help protect the security of any information you might enter into this site.
Try this
Open this in a new window

When I click 'Open in a new window' a new tab appears with the message 'Upload an image' , a grey box and a button marked 'Select an image' box , when I click this a windows file requester duly appears.

I am not sure what you mean regarding visiting and displaying URLs.
I've set the correct domain , port and subdirectory in the configuration file after seeing commit f70289f319cfbdd2e8916b41b70c88908476d93d
